src/calculateRentalCost.js

```diff
@@ -1,10 +1,19 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const ONE_DAY_COST = 40;
+
+  const totalVacationCost = days * ONE_DAY_COST;
+
+  if (days >= LONG_TERM) {
+    return totalVacationCost - LONG_TERM_DISCOUNT;
+  } else if (days >= MEDIUM_TERM) {
+    return totalVacationCost - MEDIUM_TERM_DISCOUNT;
+  } else {
```

Comment:
Avoid using `else` after an `if` statement that contains a `return`. Since the `if` block will exit the function if the condition is met, the `else` is unnecessary and can be omitted for better code readability.